Scripts/RemediationScripts/Remediate-InvalidAADObjectRoleAssignments.ps1

```diff
@@ -371,6 +371,14 @@ function Remove-AzTSInvalidAADAccounts
         Write-Host "Current user account is valid identity in AAD." -ForegroundColor Green
     }
 
+    if(($invalidClassicRoles | Measure-Object).Count -gt 0)
+    {
+        # If there is any classic deprecated assignment present for 'AccountAdministrator' role remove from the collection
+        # As AccounAdministrator assignment can't be removed using pwsh/script
+        $invalidClassicRoles = $invalidClassicRoles | Where-Object { $_.RoleDefinitionName -notlike "*AccountAdministrator*"}
+
```

Comment:
Please remove this new line. Also, if an AccountAdministrator role assignment is found, do we also add a note that one needs to reach out to Azure Support to replace this one?